python/lsst/ts/ess/ess_csc.py

```diff
@@ -203,40 +222,40 @@ async def connect_local_mode(self) -> None:
         """Connect to the ESS sensor or start the mock sensor, if in
         simulation mode.
         """
-        self.log.info("Connecting to the local sensor.")
-        self._get_device()
-        sel_temperature = SelTemperature(
-            self.config.name, self.device, self.config.channels, self.log
-        )
-        self.ess_instrument = EssInstrument(
-            self.config.name, sel_temperature, self.get_telemetry, self.log
-        )
-        self.log.info("Connection to the local sensor established.")
+        self.log.info("Connecting to the local sensor(s).")
+        for sensor_name in self.device_configurations:
+            device_configuration = self.device_configurations[sensor_name]
+            device = self._get_device(device_configuration=device_configuration)
+            sel_temperature = SelTemperature(
+                device_configuration.name,
+                device,
+                device_configuration.channels,
+                self.log,
+            )
+            self.ess_instruments.append(
+                EssInstrument(
+                    device_configuration.name,
+                    sel_temperature,
+                    self.get_telemetry,
+                    self.log,
+                )
+            )
+            self.log.info("Connection to the local sensor established.")
 
     async def connect_socket(self) -> None:
         """Connect to the SocketServer and send the configuration for which
         sensors to start reading the data of."""
         if not self.host:
-            self.host = self.config.host
+            self.host = self.config.host  # type: ignore
         if not self.port:
-            self.port = self.config.port
+            self.port = self.config.port  # type: ignore
```

Comment:
Is there some way to lose these type: ignore? Please do specify the type when you first set them in the constructor (and look for others, as well. local_mode is not specified. telemetry_loop is also not specified and unfortunately salobj has no type annotations yet. I need to get on that.